docs/resources/s3_bucket_versioning.md

```diff
@@ -0,0 +1,43 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "minio_s3_bucket_versioning Resource - terraform-provider-minio"
+subcategory: ""
+description: |-
+  
+---
+
+# minio_s3_bucket_versioning (Resource)
+
+## Example Usage
+
+```terraform
+resource "minio_s3_bucket" "bucket" {
+  bucket = "example-bucket"
+}
+
+resource "minio_s3_bucket_versioning" "bucket" {
+  bucket = minio_s3_bucket.bucket.bucket
+
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `bucket` (String)
+- `versioning_configuration` (Block List, Min: 1, Max: 1) (see [below for nested schema](#nested-schema-for-versioning_configuration))
```

Comment:
I updated this to pass validation, but I'm not sure it will work outside of GitHub's auto-generated anchor links

Reply:
That should be fine for the terraform registry as well.